starboard/raspi/shared/platform_configuration/BUILD.gn

```diff
@@ -16,6 +16,39 @@ declare_args() {
   raspi_home = getenv("RASPI_HOME")
 }
 
+config("common_flags") {
+  ldflags = [
+    "--sysroot=$raspi_home/busterroot",
+
+    # This is a quirk of Raspbian, these are required to link any GL-related
+    # libraries.
+    "-L$raspi_home/busterroot/opt/vc/lib",
+    "-Wl,-rpath=$raspi_home/busterroot/opt/vc/lib",
+    "-L$raspi_home/busterroot/usr/lib/arm-linux-gnueabihf",
+    "-Wl,-rpath=$raspi_home/busterroot/usr/lib/arm-linux-gnueabihf",
+    "-L$raspi_home/busterroot/lib/arm-linux-gnueabihf",
+    "-Wl,-rpath=$raspi_home/busterroot/lib/arm-linux-gnueabihf",
+
+    # Cleanup unused sections
+    "-Wl,-gc-sections",
+    "-Wl,--unresolved-symbols=ignore-all",
```

Comment:
i think we need to clean up all these unresolved symbols in alter cl - this apaprently is hard to do for ps4

@kaidokert  can you ptal

Reply:
this shouldnt block this PR IMO , so feel free to resolve